celeri.py

```diff
@@ -2237,7 +2237,7 @@ def test_okada_equals_cutde():
     # Parameters for Okada
     segment_locking_depth = 1.0
     segment_dip = 90
-    segment_length = 0.5
+    segment_length = 1.0
```

Comment:
Since this length is divided by two below, it should be `1.0` to match with the `[0.5, 0.5]` x range for the triangles.

Reply:
I understand this better now.  It looks like I somehow got it right in the segment case.